content/en/docs/tasks/configure-pod-container/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Configure Pods and Containers"
 description: Perform common configuration tasks for Pods and containers.
-weight: 20
+weight: 30
```

Comment:
I may not be understanding the issue-at-hand correctly here, so please feel free to correct me if I'm wrong. The Index page for a particular section is expected to have a higher page weight than the ones below it or not having the same width? However content/en/docs/tasks/configure-pod-container/configure-gmsa.md and this one have the same page weight?

Reply:
@divya-mohan0209 The /_index.md file determines how a folder shows up in the side nav, i believe its a separate ordering set from the single .md file page weight inside a folder. So the page weight of 30 for the _index.md file makes sure Configure Pods and Containers shows up 3rd in the list under the Task section. Because Configure Pods and Containers is also a folder, when Hugo builds the side nav for the items in the folder, it resets the page weight to 0 then orders the list of pages under Configure Pods and Containers. 

The 30 in content/en/docs/tasks/configure-pod-container/configure-gmsa.md makes sure that file shows up 3rd under Configure Pods and Containers in the side nav.

Reply:
Cool, thank you so much for clarifying 💜